x/evm/statedb/config.go

```diff
@@ -46,4 +46,5 @@ type EVMConfig struct {
 	ChainConfig *params.ChainConfig
 	CoinBase    common.Address
 	BaseFee     *big.Int
+    WhitelistAddresses []string
```

Comment:
this is not needed since you already have this in the `Params`

Reply:
This is to avoid quering the store twice for performance.

Reply:
nvm you are right - I just saw what you meant!